app/Http/Controllers/LanguageController.php

```diff
@@ -75,9 +78,15 @@ public function edit(Language $language)
      */
     public function update(LanguageRequest $request, Language $language)
     {
-        $language->update($request->all());
-        return redirect(route('languages.index'))
-            ->with(session()->flash('success','Language is updated successfully .'));;
+        if (Language::where('name', $request->name)->where('id', '!=', $language->id)->count() == 0) {
+            Language::where('id', $language->id)->update(['name' => $request->name]);
+             return redirect()->route('languages.index')
+                        ->with(session()->flash('success','language is Updated successfully .'));
+        }
+        else{
+            $err['name']='This is already exist';
```

Comment:
Rename it to
This language already exists.

Reply:
done